### PR TITLE
Reshuffle docs/ versions/ and ush/ contents - release/bufr_tranjb.v1.0.2_RM

### PR DESCRIPTION
**Pull request by ilianagenkova**
-move build.sh from top level dir to ush/
-move VERSION  to versions/
-made docs/ directory
-moved README.md to docs/
-added Release_Notes.txt (from wcoss2 transition)
-updated CMakeLists.txt, build.sh and README.md

I couldn't not find Mitch Weiss's user name, if anyone knows it, please tag Mitch for awareness/input. thanks!

### CODE REVIEW

File: ush/CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 list(APPEND ush_files
+  build.sh
```
Comment (aerorahul) [inline ush/CMakeLists.txt]: this file should not need be installed.

**Review by aerorahul — CHANGES_REQUESTED** [review]
just 1 change requested. the rest looks fine.

**Comment by aerorahul** [comment]
@ilianagenkova 
Please
- resolve the conflicts in the PR
- apply the requested changes in the prior review

**Comment by ilianagenkova** [comment]
@aerorahul done!

File: ush/CMakeLists.txt (hunk 1 above)
Comment (ilianagenkova) [reply ush/CMakeLists.txt]: After correcting it in the wrong branch first, I did it right the second time around. thanks!

**Pull request merged**